android/app/src/main/java/mqttController/CarConnect.java

```diff
@@ -73,7 +74,9 @@ public void onSuccess(IMqttToken asyncActionToken) {
                     isConnected = true;
                     final String successfulConnection = "Connected to MQTT broker";
                     Log.i(TAG, successfulConnection);
-                    Toast.makeText(context, successfulConnection, Toast.LENGTH_SHORT).show();
+                    Toast toast = Toast.makeText(context, successfulConnection, Toast.LENGTH_SHORT);
+                    toast.setGravity(Gravity.TOP,0,0);
+                    toast.show();
```

Comment:
I think it would be better to put this into a method, and then call this method when it is needed.